prow/overlays/cnv-prod/peribolos.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: batch/v1
+kind: Job
```

Comment:
@goern shouldn't this be a cron job? Do we plan to get it synced on config change? If yes and it would react on config update, we can use peribolos to create new repos/invite to org via PR and on demand.

Reply:
I thought of it as a Job so that we have more control over it during experiment/testing

Reply:
I think it should be a prow job that is run after merge to master (in this case) whenever the `org_*` file changes